### PR TITLE
adapt capacitive sensing to board and add Segger RTT

### PR DESCRIPTION
**Pull request by atsju**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -25,42 +27,115 @@ int main(void)
...
+        }
+
+
+
+        for(uint8_t i=0; i<NB_CAP_CHAN/2; i++)
+        {
+            // one line and one colums value per screen line
+            sprintf(printBuffer, "%1i %4i %4i", i, meanMeas[i]-calibs[i], meanMeas[i+NB_CAP_CHAN/2]-calibs[i+NB_CAP_CHAN/2]);
+            sharpMemoryLCD_printTextLine(i, printBuffer, 11);
+            HAL_Delay(10);
```
Comment (Hish15) [inline main.c:125]: Why do you need this delay?

Comment (atsju) [reply main.c:125]: everything is not working smooth with the screen. Some bugs to solve ... Later

Comment (Hish15) [reply main.c:125]: You could put the quickfix into the problematic function?

Comment (atsju) [reply main.c:125]: Yes...

**Pull request merged**